draft-ietf-core-object-security.md

```diff
@@ -2055,7 +2061,7 @@ This section lists and discusses issues with unprotected message fields.
 * Uri-Host/Uri-Port. In forward proxy deployments, the Uri-Host/Uri-Port may be changed by an adversary, and the application needs to handle the consequences of that (see {{uri-host}}). 
 The Uri-Host may either be omitted, reveal information equivalent to that of the IP address or more privacy-sensitive information, which is discouraged.
 
-* Observe. The Outer Observe option is intended for an OSCORE-unaware proxy to support forwarding of Observe messages, but is ignored by the endpoints since the Inner Observe determines the processing in the endpoints. Since the Partial IV provides absolute ordering of notifications it is not possible for an intermediary to spoof reordering (see {{observe}}). The size and distributions of notifications over time may reveal information about the content or nature of the notifications. 
+* Observe. The Outer Observe option is intended for an OSCORE-unaware proxy to support forwarding of Observe messages, but is ignored by the endpoints since the Inner Observe determines the processing in the endpoints. Since the Partial IV provides absolute ordering of notifications it is not possible for an intermediary to spoof reordering (see {{observe}}). The absence of Partial IV, since only allowed for the first notification, does not prevent correct ordering of notifications. The size and distributions of notifications over time may reveal information about the content or nature of the notifications.
 
```

Comment:
This is for all proxies to support forwarding of Observe messages.  Does not matter if they are OSCORE aware or not.

Reply:
The reasoning behind this "OSCORE-unaware" was the following: OSCORE aware proxies should know that observe might be hidden from the message, and may be set up to always fw messages with OSCORE (in which case outer observe is not needed), even when receiving several responses to the same request, while unaware proxies would just drop those additional messages. Do you think we should just remove "OSCORE-unaware"?

Reply:
If you are going to deal with things that way then you need to distinguish between the Observe option when in a request and when in a response for an OSCORE aware proxy.  Specifically, an OSCORE aware proxy could keep the routing open speculatively when a response w/o the option being present, but you end up with the question of how a server would know if the proxies in the system are or are not OSCORE aware.  Keeping thing consistent unless there is a big benefit seems to be a better idea.